xml/rmt_mirroring.xml

```diff
@@ -160,19 +161,28 @@ Jun 22 04:22:34 d31 systemd[1]: Started RMT Mirror timer.</screen>
     Example:
    </para>
 <screen>&prompt.sudo;<command>rmt-cli products list --all</command>
-+------+------------------+---------+--------------+----------------------+---------------+---------+---------------+
-| ID   | Name             | Version | Architecture | Product string       | Release stage | Mirror? | Last mirrored |
-+------+------------------+---------+--------------+----------------------+---------------+---------+---------------+
++------+------------------+---------+--------------+----------------------+
```

Comment:
I just double-checked and ran `rmt-cli products list --all` on our reference installation of RMT (which is always on the latest version) and the output of this command has slightly changed:

```
root@rmt:~ # rmt-cli products list --all                                                                                                                                                                                               
+------+-------------------------------------------------------------------------+---------+---------+--------------+-------------------------+                                                                                                
| ID   | Product                                                                 | Version | Arch    | Mirror?      | Last mirrored           |                                                                                                
+------+-------------------------------------------------------------------------+---------+---------+--------------+-------------------------+                                                                                                
| 1294 | Advanced Systems Management Module                                      | 12      | ppc64le | Don't Mirror |                         |                                                                                                
|      | sle-module-adv-systems-management/12/ppc64le                            |         |         |              |                         |                                                                                                
| 1295 | Advanced Systems Management Module                                      | 12      | s390x   | Don't Mirror |                         |                                                                                                
|      | sle-module-adv-systems-management/12/s390x                              |         |         |              |                         |                                                                                                
| 1212 | Advanced Systems Management Module                                      | 12      | x86_64  | Mirror       | 2020-01-28 03:24:41 UTC |                                                                                                
|      | sle-module-adv-systems-management/12/x86_64                             |         |         |              |                         |                                                                                                
| 1589 | Basesystem Module                                                       | 15      | aarch64 | Don't Mirror |                         |                                                                                                
|      | sle-module-basesystem/15/aarch64                                        |         |         |              |                         |                                                                                                
```

And so on.

Have you left out some columns on purpose? That's OK I guess, but please note that there is no column "Product string" anymore, but that we display it in the first column below the full name now.

Reply:
fixed